src/Mapbender/PrintBundle/Component/PrintService.php

```diff
@@ -456,13 +456,13 @@ private function buildPdf()
 
                         // fill digitizer feature fields
                         if(preg_match("/^feature./", $k)){
-                            if($feature == false){
+                            if(!isset($feature) OR $feature == false){
                                 continue;
                             }
                             $attribute = substr(strrchr($k, "."), 1);
                             $pdf->MultiCell($this->conf['fields'][$k]['width'],
                                 $this->conf['fields'][$k]['height'],
-                                $feature->getAttribute($attribute));
+                                utf8_decode($feature->getAttribute($attribute)));
```

Comment:
@astroidex, to save decoded `$attrubute` you need to assign new value into it:
`$attribute =  utf8_decode($feature->getAttribute($attribute));`

Do it's rightly worked before? 
Other thing is: one `)` sign more, at eol, as expected.